packages/coil-extension/src/popup/components/Unsubscribed.tsx

```diff
@@ -7,10 +7,10 @@ import { PopupProps } from '../types'
 import { StatusButton } from './StatusButton'
 import { StatusTypography } from './util/StatusTypography'
 
-const titleString = 'Get a subscription'
-const subheading1 = 'To use Coil you need an active subscription'
+const titleString = 'Get a membership'
```

Comment:
This seems kind of awkward. Is the plan to use membership on the site as well?

Reply:
![image](https://user-images.githubusercontent.com/525211/72214088-99e5dc00-352d-11ea-88ae-a274496d3023.png)

^ This will become Membership too ?

Reply:
The site will change over to membership once the new settings page Dacio is working on is merged in.